Cargo.toml

```diff
@@ -72,7 +95,7 @@ license = "MIT"
 name = "languagetool-rust"
 readme = "README.md"
 repository = "https://github.com/jeertmans/languagetool-rust"
-rust-version = "1.77.2"
+rust-version = "1.80.0"
```

Comment:
Do you know what package required this bump?

Reply:
That was the usage of `LazyLock` in `tests/cli.rs`

Reply:
Humm, `dev-dependencies` shouldn't affect the MSRV in my opinion. There's probably a work around for this.

Reply:
I can just avoid using `LazyLock` if you prefer, and build the paths for each test

Reply:
Or there is probably another option that does not require bumping the MSRV :) I will give it a look

Reply:
Done: 4fa1244584765fe41ad93b04935b08b7452ab543